wbooks/src/redux/book/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import { BOOKS } from '@constants/types';
+
+import { actionCreators } from './actions';
+
+const initialState = {
+  ok: false,
+  data: []
+};
+
+interface Action {
+  type: string;
+}
+
+export const bookReducer = (state = initialState, action: Action) => {
+  switch (action.type) {
+    case BOOKS.GET:
+      return {
+        ok: true,
+        data: actionCreators.getBooks()
+      };
```

Comment:
No, in the `GET_BOOK` you must set `booksLoading: true`
In `GET_BOOKS_SUCCESS` set `booksLoading: false` and `books: action.payload` and you forgot `GET_BOOK_FAILURE` and save `booksError: action.payload` 

Remember that state is an object